### PR TITLE
Add UI support for deleting and renaming applications

### PR DESCRIPTION
**Pull request by jrgp**
Both actions only available to admins

### CODE REVIEW

File: src/iris/api.py [hunk 1]
```diff
@@ -1858,6 +1858,26 @@ def on_put(self, req, resp, app_name):
...
+            session.commit()
+        except IntegrityError:
+            raise HTTPBadRequest('Cannot remove app. It has likely already in use.', '')
+        finally:
+            session.close()
+
+        if not affected:
+            raise HTTPBadRequest('No rows changed; app name probably already deleted', '')
+
+        resp.status = HTTP_204
```
Comment (houqp) [inline src/iris/api.py]: let's keep the status code consistent with the other apis, i.e. return 200 if body is not empty.

Comment (jrgp) [inline src/iris/api.py]: Fixed this

**Comment by houqp** [comment]
WTF, why are you pushing code at 1:30 am.

**Comment by jrgp** [comment]
Because I can xD

**Pull request merged**